packages/appkit-utils/src/ConstantsUtil.ts

```diff
@@ -9,13 +9,20 @@ export const ConstantsUtil = {
   SAFE_CONNECTOR_ID: 'safe',
   LEDGER_CONNECTOR_ID: 'ledger',
   EIP6963_CONNECTOR_ID: 'eip6963',
-  AUTH_CONNECTOR_ID: 'w3mAuth',
+  AUTH_CONNECTOR_ID: 'ID_AUTH',
```

Comment:
why change this?

Reply:
It was interfering with the providerType, so I think this is better to distinguish type and id